boards/nucleo_f429zi/src/main.rs

```diff
@@ -302,98 +302,98 @@ pub unsafe fn reset_handler() {
+            2 => stm32f429zi::gpio::PIN[5][15].as_ref().unwrap(), //D2
+            3 => stm32f429zi::gpio::PIN[4][13].as_ref().unwrap(), //D3
+            4 => stm32f429zi::gpio::PIN[5][14].as_ref().unwrap(), //D4
+            5 => stm32f429zi::gpio::PIN[4][11].as_ref().unwrap(), //D5
+            6 => stm32f429zi::gpio::PIN[4][9].as_ref().unwrap(), //D6
+            7 => stm32f429zi::gpio::PIN[5][13].as_ref().unwrap(), //D7
+            8 => stm32f429zi::gpio::PIN[5][12].as_ref().unwrap(), //D8
+            9 => stm32f429zi::gpio::PIN[3][15].as_ref().unwrap(), //D9
             // SPI Pins
-            stm32f429zi::gpio::PIN[3][14].as_ref().unwrap(), //D10
-            stm32f429zi::gpio::PIN[0][7].as_ref().unwrap(),  //D11
-            stm32f429zi::gpio::PIN[0][6].as_ref().unwrap(),  //D12
-            stm32f429zi::gpio::PIN[0][5].as_ref().unwrap(),  //D13
+            10 => stm32f429zi::gpio::PIN[3][14].as_ref().unwrap(), //D10
+            11 => stm32f429zi::gpio::PIN[0][7].as_ref().unwrap(),  //D11
+            12 => stm32f429zi::gpio::PIN[0][6].as_ref().unwrap(),  //D12
+            13 => stm32f429zi::gpio::PIN[0][5].as_ref().unwrap(),  //D13
             // I2C Pins
-            stm32f429zi::gpio::PIN[1][9].as_ref().unwrap(), //D14
-            stm32f429zi::gpio::PIN[1][8].as_ref().unwrap(), //D15
-            stm32f429zi::gpio::PIN[2][6].as_ref().unwrap(), //D16
-            stm32f429zi::gpio::PIN[1][15].as_ref().unwrap(), //D17
-            stm32f429zi::gpio::PIN[1][13].as_ref().unwrap(), //D18
-            stm32f429zi::gpio::PIN[1][12].as_ref().unwrap(), //D19
-            stm32f429zi::gpio::PIN[0][15].as_ref().unwrap(), //D20
-            stm32f429zi::gpio::PIN[2][7].as_ref().unwrap(), //D21
+            14 => stm32f429zi::gpio::PIN[1][9].as_ref().unwrap(), //D14
+            15 => stm32f429zi::gpio::PIN[1][8].as_ref().unwrap(), //D15
+            16 => stm32f429zi::gpio::PIN[2][6].as_ref().unwrap(), //D16
+            17 => stm32f429zi::gpio::PIN[1][15].as_ref().unwrap(), //D17
+            18 => stm32f429zi::gpio::PIN[1][13].as_ref().unwrap(), //D18
+            19 => stm32f429zi::gpio::PIN[1][12].as_ref().unwrap(), //D19
+            20 => stm32f429zi::gpio::PIN[0][15].as_ref().unwrap(), //D20
+            21 => stm32f429zi::gpio::PIN[2][7].as_ref().unwrap(), //D21
             // SPI B Pins
-            stm32f429zi::gpio::PIN[1][5].as_ref().unwrap(), //D22
-            stm32f429zi::gpio::PIN[1][3].as_ref().unwrap(), //D23
-            stm32f429zi::gpio::PIN[0][4].as_ref().unwrap(), //D24
-            stm32f429zi::gpio::PIN[1][4].as_ref().unwrap(), //D25
+            // None => 22, // Some => stm32f429zi::gpio::PIN[1][5].as_ref().unwrap(), //D22
+            // None => 23, // Some => stm32f429zi::gpio::PIN[1][3].as_ref().unwrap(), //D23
+            // None => 24, // Some => stm32f429zi::gpio::PIN[0][4].as_ref().unwrap(), //D24
+            // None => 25, // Some => stm32f429zi::gpio::PIN[1][4].as_ref().unwrap(), //D25
```

Comment:
Probably best to remove the "None" portions of these comments since that is not part of the interface anymore

Reply:
Done